password.py

```diff
@@ -1,25 +1,60 @@
 #import
 
+#dictionaries for storing accounts,and passwords
 account_dict = {}
-sec_questions_dict = {}
+special_characters = ['!','@','#','$','%','&','*']
+i = 0
 
 def account_create():
-    print("account created")
+    print("-"*18, "\nAccount Creation!"), print("-"*18)
+    username = input("Enter a Username:\n")
+    password = input("Enter a Password:\n")
+    if len(password) < 6:
+        print("Password has to be 6 characters or more.")
+    print(security_questions())
+    security = input("Please choose a security question (Type in corresponding number):\n")
+    
 
 
+    #fail parameters
+    if len(password) < 6:
+        print("Password has to be 6 characters or more.")
+    
+    #elif password != i:
+        #while True:
+            #i+1
+            #if password == special_characters[i]:
+                #break
+            #elif password != special_characters[i]:
+                #print("Password has to have atleast 1 special character")
+                #break
+    #elif password < numbers in range(11):
+        #print("password has to have a number")
+    
+
 def security_questions():
-    print("security questioned")
+    sec_questions_dict = {
+        "1":"What's your dog's name",
+        "2":"What's your cat's name",
+        "3":"What's your best friend's middle name",
+        "4":"some fourth option"
+     }
+    return
+
+def password_manager():
+    print("some third thing")
 
 
 def main():
-    print("Hey welcome to password manager!")
-    answer = input("Do you have an account? Type 'yes' or 'no': ")
+    print("Hey welcome to Password Manager!")
+    answer = input("Do want to create an account? Type 'yes' or 'no': ")
     accepted_answers = ['yes','no']
 
     if answer == accepted_answers[0]:
```

Comment:
You can just make this if `answer == "yes"`. Try not to use lists when you're trying to pull a specific value from a list. Dictionaries are better for that and more straightforward. But you don't need to use either here since the values are only yes and no.